src/colorizer/utils/math_utils.ts

```diff
@@ -9,7 +9,7 @@ export function numberToStringDecimal(
   decimalPlaces: number,
```

Comment:
I can't make a suggestion because this line is a little too far up, but also should switch the type of `input` to explicitly accept `null` types here too.